engine/rowset/src/main/java/io/deephaven/engine/rowset/RowSetBuilderSequential.java

```diff
@@ -43,7 +43,11 @@ default void appendKeys(PrimitiveIterator.OfLong it) {
     }
 
     default void appendOrderedRowKeysChunk(final LongChunk<OrderedRowKeys> chunk) {
-        appendKeys(new LongChunkIterator(chunk));
+        appendOrderedRowKeysChunk(chunk, 0, chunk.size());
+    }
+
+    default void appendOrderedRowKeysChunk(final LongChunk<OrderedRowKeys> chunk, int offset, int length) {
```

Comment:
Are we missing an overload for `addOrderedRowKeysChunk` with `OrderedRowKeys` to map to append?